hazelcast/projection.py

```diff
@@ -49,9 +51,8 @@ def get_class_id(self):
         return 0
 
 
-class _MultiAttributeProjection(_AbstractProjection):
-    def __init__(self, attribute_paths):
-        # type: (list[str]) -> None
+class _MultiAttributeProjection(_AbstractProjection[typing.List[typing.Any]]):
```

Comment:
`typing.List[typing.Any]` can be simplified to `typing.List` or just `list`. There are more.